src/components/OwnerList.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from 'react';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  TablePagination,
+} from '@material-ui/core';
+import { findAllOwners } from '../service/PetClinicService';
+
+const Owners = () => {
+  const [owners, setOwners] = useState([
+    { id: '', firstName: '', lastName: '', address: '', city: '', phone: '' },
+  ]);
+
+  useEffect(() => {
+    findAllOwners()
+      .then((data) => {
+        setOwners(data);
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+  }, []);
+
+  const [page, setPage] = useState(0);
```

Comment:
For better readability, I suggest to group `useState()` hooks together like this

```tsx
const [owners, setOwners] = useState([{ id: '', firstName: '', lastName: '', address: '', city: '', phone: '' }]);
const [page, setPage] = useState(0);
const [rowsPerPage, setRowsPerPage] = useState(8);

// then useEffect() ...
```

Also, use typing like this 
```tsx
const [page, setPage] = useState<number>(0);
```

Reply:
Note: Will be done later